src/components/Button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon } from '../Icon/Icon';
+
+import './Button.scss';
+
+export const Button = ({
+  toggle,
+  isActive,
+  color,
+  iconName,
+  iconColor,
```

Comment:
You can simplify the Button appearance using one prop instead of 2 (`color` and `iconColor`) since there are only 2 visual variants of the Button: primary (red with white text) or secondary (gray with dusk text).